the-journal-of-egyptian-archaeology.csl

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>The Journal of Egyptian Archaeology</title>
+    <id>http://www.zotero.org/styles/the-journal-of-egyptian-archaeology</id>
+    <link href="http://www.zotero.org/styles/the-journal-of-egyptian-archaeology" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-the-history-of-collections" rel="template"/>
+    <link href="http://www.ees.ac.uk/publications/journal-egyptian-archaeology.html" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <issn>0075-4234</issn>
+    <eissn>0307-5133</eissn>
```

Comment:
(Unless you have answers about this genuine mystery, 🤔, I would just use 0307-5133 as the print ISSN, per https://www.ncbi.nlm.nih.gov/nlmcatalog/?term=0075-4234, and leave out the other one)